create.rb

```diff
@@ -0,0 +1,112 @@
+class CreatePeople
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'Database is empty! Add a book.' if @books.empty?
+    @books.each { |book| puts "[Book] Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_all_persons
+    puts 'Database is empty! Add a person.' if @persons.empty?
+    @persons.each do |person|
+      puts "[#{person.class.name}] Name: #{person.name}, Age: #{person.age}, id: #{person.id}"
+    end
+  end
+
+  def list_all_rentals
+    puts 'To see rentals enter the person ID: '
+    id = gets.chomp.to_i
+
+    puts 'Rented Books: '
+    @rentals.each do |rental|
+      if rental.person.id == id
+        puts "Person: #{rental.person.name} Date: #{rental.date}, Book: '#{rental.book.title}' by #{rental.book.author}"
+      else
+        puts
+        puts 'No record were found for the given ID'
+      end
+    end
+  end
+
+  def create_person
+    print 'press 1 to create a student, press 2 to create a teacher : '
+    option = gets.chomp
+
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Invalid input. Try again'
+    end
+  end
+
+  def create_student
+    puts 'create a new student'
+    print 'Enter student age: '
+    age = gets.chomp.to_i
+    print 'Enter name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase
```

Comment:
- [ ] Kindly do the same thing with the input handling here and all preceding functions aswell.